scripts/mocha-runner.js

```diff
@@ -12,7 +12,7 @@ const options = {
 
 let mocha = new Mocha({...options})
 
-glob.sync('src/**/*spec.ts').forEach((file) => {
+glob.sync('src/**/*spec.tsx').forEach((file) => {
```

Comment:
not each file will be tsx, so the pattern should actually include both. You'll have enzyme component tests and also unit testing in some of the cases

Reply:
something like 'src/**/*spec.ts?(x)' or 'src/**/*spec.ts?x'

You can look at the pattern conventions here

Reply:
https://www.npmjs.com/package/glob